crates/typst/src/math/cancel.rs

```diff
@@ -65,13 +65,13 @@ pub struct CancelElem {
 
     /// How much to rotate the cancel line.
     ///
-    /// - If `{auto}`, the line assumes the default angle; that is, along the
-    ///   diagonal line of the content box.
     /// - If given an angle, the line is rotated by that angle clockwise w.r.t
```

Comment:
I think `w.r.t` should also be expanded to `with respect to`.

Reply:
I used it since
- that's what was used originally
- doesn't seem that obscure of an initialism
- changing it here would essentially make the way of writing different from all the other places this initialism is used, and, like, we might want to open a separate issue for something like that(?)

,what do you think?

(ESL, so apologies if any sentence structure seems awkward)

Reply:
I would definitely write it out in full.

Reply:
Should I open a separate issue for that or just lump the changes with this one

Reply:
Do you have examples of other instances of "w.r.t." or other abbreviations in the documentation?

Reply:
![Screenshot_2024-06-25-01-31-32-862_com.github.android-edit.jpg](https://github.com/typst/typst/assets/74085528/5e80504b-a6de-4841-a8d4-c1954544f007)

Apart from the file changed for this pr, these are other two using w.r.t. away from my pc currently but I will try to run a search for other abbreviations if needed, when I get back

Reply:
I prefer to write "with respect to" fully.

I believe if the document is written for users, especially the API reference, it should be readable and easy to understand for everyone (including non-native English users & inexperienced developers).

BTW, here is the documentation for users, whereas the "w.r.t" in your posted image is for developers, but not for users.

Reply:
I see. I'll change it to the full words